mono/metadata/appdomain.c

```diff
@@ -209,6 +210,14 @@ create_domain_objects (MonoDomain *domain)
 	mono_field_static_set_value (string_vt, string_empty_fld, empty_str);
 	domain->empty_string = empty_str;
 
+	bitconverter_isle_fld = mono_class_get_field_from_name_full (mono_defaults.bitconverter_class, "IsLittleEndian", NULL);
+	if (bitconverter_isle_fld) {
+		gint x = 1;
+		MonoBoolean is_le = *((char*) &x); // 00 00 00 01 for little-endian and 01 00 00 00 for big-endian
```

Comment:
Just use the TRUE/FALSE constants.

Reply:
x should be const/static. Or use G_ENDIAN.